app/src/main/java/com/odoo/followup/LoginActivity.java

```diff
@@ -48,37 +50,36 @@ private void init() {
     @Override
     public void onClick(View view) {
         mView = view;
-        if (isValid()) {
-            try {
-                odoo = Odoo.createInstance(this, hostURL());
-                odoo.setOnConnect(this);
-            } catch (OdooVersionException e) {
-                e.printStackTrace();
-            }
-
+        isValid();
+        try {
+            odoo = Odoo.createInstance(this, hostURL());
+            odoo.setOnConnect(this);
+        } catch (OdooVersionException e) {
+            e.printStackTrace();
```

Comment:
Show warning dialog (Alertdialog) for invalid version support and put warning log also.